### PR TITLE
Merge hotfixes put into staging back down to develop

### PR DESCRIPTION
**Pull request by briehl**
Kind of a pain, but there's lots of conflicts here that need to get resolved before our next deploy this week.

Tagging @msneddon @scanon @kkellerlbl @eapearson @rsutormin - please test that I didn't do anything completely insane

### CODE REVIEW

**Comment by briehl** [comment]
Also, just a note that I did this merge on my fork with the latest KBase fork of staging, and worked out issues there. Back end tests pass as expected, but we're using Travis-CI for front end unit tests.

**Comment by kkellerlbl** [comment]
If I understand this right, are these changes already in the staging branch?  Are there additional changes from develop that we want to merge up to staging this week?

File: docker/docker.lua [hunk 1]
```diff
@@ -43,133 +43,136 @@ end
...
-					     "search_images" : { 
-						"path" : "/images/search",
-						"method" : "GET",
-						"required_params" : [
-						   "term"
-						],
-					     },
-					  }
-				       }]]
+	"base_url" : 'http://127.0.0.1:65000',
```
Comment (kkellerlbl) [inline docker/docker.lua]: The spacing on this file changed, so it's a bit hard to tell exactly what's changed.  But it looks ok at first glance.

Comment (briehl) [inline docker/docker.lua]: The spacing should be the only fix - I think that was just my editor making things readable.

**Comment by briehl** [comment]
Yes, this is just to get the develop branch caught up with all the changes we made directly to staging. It's the first step toward merging all of develop (with all the appropriate changes) up into staging. But that's not happening yet.

**Pull request merged**